python/lsst/afw/table/aggregates/aggregatesContinued.py

```diff
@@ -0,0 +1,53 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["PointKey", "CovarianceMatrixKey"]
+
+from future.utils import with_metaclass
+import numpy as np
+
+from lsst.utils import TemplateMeta
+from . import aggregates
+
+
+class PointKey(with_metaclass(TemplateMeta, object)):
+    pass
+
+
+PointKey.register(np.int32, aggregates.Point2IKey)
+PointKey.register(np.float64, aggregates.Point2DKey)
```

Comment:
Doesn't this exclude the future possibility of having a `Point3IKey` (by dropping the `2` in the abc)?